package.json

```diff
@@ -10,7 +10,8 @@
   "scripts": {
     "build": "webpack --config webpack.config.js --mode development --watch",
```

Comment:
I would recommend changing this to remove the watch and dev or creating two different commands.One for dev which can include the --watch and one for standard build which does not include --watch

Reply:
Great idea.